middleman-core/fixtures/env-app/source/stylesheets/site.css.scss

```diff
@@ -1,3 +1,3 @@
 body {
-  backgroud: red;
+  background: red;
```

Comment:
This is either a bugfix or an intentional bug that should be commented.

Reply:
This is from a test, so if the tests still pass after changing, it should be fine.